web_external/templates/body/submissionPage.jade

```diff
@@ -18,7 +18,7 @@
   if submission.has('overallScore')
     .c-submission-overall-score-container
       .c-pre-text Overall score:
-      .c-overall-score= Math.round(submission.get('overallScore') * 1000) / 1000
+      .c-overall-score= submission.get('overallScore').toFixed(3)
```

Comment:
Note that this is not equivalent, if the score is 100, this will now show `100.000` where before it would show just `100`. Is that an intentional behavior change?